xbmc/osx/XBMCHelper.cpp

```diff
@@ -114,7 +114,7 @@ bool XBMCHelper::OnSettingChanging(const CSetting *setting)
     if (IsRunning() && GetMode() != remoteMode)
     {
       bool cancelled;
-      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, 13146, 13147, -1, -1, cancelled, 10000))
+      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, cancelled, -1, -1, 10000))
```

Comment:
the YesNo dialog has 6 ways to be called. Each with a different parameter order and some even with a different parameter type. In this case, we're using the method that directly accepts a "text" variable and doesn't have any line1, line2, line3 handling, and this particular method overload is expecting a string to be passed for the alternate YES/NO labels instead of integers (-1) like the line1/2/3 version of this method does. Any further questions? :)